.config/.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@naguib-med please update this file to include this instead. I have a feeling the current setup will break the UI:

{
    "printWidth": 10000,
    "overrides": [
      {
        "files": "*.md",
        "options": {
          "parser": "markdown"
        }
      }
    ]
  }